### PR TITLE
Move gd functions to gd module file

### PR DESCRIPTION
**Pull request by b1rdex**

### CODE REVIEW

File: gd/gd.php [hunk 1]
```diff
@@ -2927,4 +2927,418 @@ function imagebmp ($image, $to = null, $compressed = true) {}
...
+ * @return bool Returns TRUE on success or FALSE on failure.
+ * @since 5.5.0
+ */
+function imagesetinterpolation ($image, $method = IMG_BILINEAR_FIXED) {}
+
+/**
+ * Get Mime-Type for image-type returned by getimagesize,
+ * @since 4.3.0
+ * @since 5.0
+exif_read_data, exif_thumbnail, exif_imagetype
```
Comment (King2500) [inline gd/gd.php]: ??

Comment (b1rdex) [reply gd/gd.php]: Ty, fixed. Original phpdoc was broken already.

File: gd/gd.php [hunk 2]
```diff
@@ -2927,4 +2927,570 @@ function imagebmp ($image, $to = null, $compressed = true) {}
...
+ * Index 3 is a text string with the correct
+ * height="yyy" width="xxx" string that can be used
+ * directly in an IMG tag.
+ * </p>
+ * <p>
+ * mime is the correspondant MIME type of the image.
+ * This information can be used to deliver images with correct the HTTP
+ * Content-type header:
+ * getimagesize and MIME types
+ * ]]>
```
Comment (pestretsov) [inline gd/gd.php]: Please, also remove this formatting leftover

**Comment by b1rdex** [comment]
@pestretsov I moved iptcparse and iptcembed back as there is note on php.net that it doesn't require GD exteinsion.

**Comment by pestretsov** [comment]
@b1rdex thank you very much, I've almost missed that. You are completely correct about `iptcparse` and `iptcembed`, but it seems like there are more such functions: [`getimagesize`](https://www.php.net/manual/en/function.getimagesize.php#refsect1-function.getimagesize-notes), [`image_type_to_extension`](https://www.php.net/manual/en/function.image-type-to-extension.php#refsect1-function.image-type-to-extension-notes), [`image_type_to_mime_type`](https://www.php.net/manual/en/function.image-type-to-mime-type.php#refsect1-function.image-type-to-mime-type-notes). Could you, please, also move those functions back with the related constants.

**Comment by b1rdex** [comment]
@pestretsov Ok I moved them back and IMAGETYPE_* constants too as they're in core too: https://github.com/php/php-src/blob/879cd0491399ccfacac0d6ed701d998a65a6cc97/ext/standard/image.c#L73

**Comment by pestretsov** [comment]
@b1rdex thank you very much, you've done an awesome job!

**Pull request merged**